src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/UnpaidAttendanceSummaryReportITest.java

```diff
@@ -47,6 +44,7 @@ protected StandardReportRequest getValidPayload() {
     }
 
     @Test
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
```

Comment:
This should be at class level instead of method level

Reply:
updated

Reply:
Can we add some valid data into the sql that is outside the new range to ensure it does not get picked up?

Reply:
updated :-)